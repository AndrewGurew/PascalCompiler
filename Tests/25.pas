if x > 0 then modul := x else modul := -x;
if x>0 then modul:=x else modul:=-x;