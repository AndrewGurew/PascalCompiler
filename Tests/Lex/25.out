Position            Type                     Text                Value               
-------------------------------------------------------------------------------------
(1,1)               If                       if                  if                  
(1,4)               Id                       x                   x                   
(1,6)               More                     >                   >                   
(1,8)               Integer                  0                   0                   
(1,10)              Id                       then                then                
(1,15)              Id                       modul               modul               
(1,21)              Assign                   :=                  :=                  
(1,24)              Id                       x                   x                   
(1,26)              Else                     else                else                
(1,31)              Id                       modul               modul               
(1,37)              Assign                   :=                  :=                  
(1,40)              Minus                    -                   -                   
(1,41)              Id                       x                   x                   
(1,42)              Semicolon                ;                   ;                   
(2,2)               If                       if                  if                  
(2,5)               Id                       x                   x                   
(2,6)               More                     >                   >                   
(2,7)               Integer                  0                   0                   
(2,9)               Id                       then                then                
(2,14)              Id                       modul               modul               
(2,19)              Assign                   :=                  :=                  
(2,21)              Id                       x                   x                   
(2,23)              Else                     else                else                
(2,28)              Id                       modul               modul               
(2,33)              Assign                   :=                  :=                  
(2,35)              Minus                    -                   -                   
(2,36)              Id                       x                   x                   
(2,37)              Semicolon                ;                   ;                   
(2,37)              End of file              EndOfFile           EndOfFile           
